packages/teams-js/src/internal/communication.ts

```diff
@@ -52,6 +52,9 @@ class CommunicationPrivate {
   public static promiseCallbacks: {
     [id: number]: Function; // (args[]) => void
   } = {};
+  public static portCallbacks: {
+    [id: number]: Function; // (args[], port) => void
```

Comment:
>Function; // (args[], port) => void [](http://example.com/codeflow?start=18&length=35)

Rather than declaring the value as type `Function` with the expectation in the comment (are the expected parameters reversed in the comment?), could this be declaration be more explicit about the expected function type, e.g.,

```
public static portCallbacks: {
  [id: number]: (port: MessagePort, args?: unknown[]) => void;
} = {};
```

Reply:
That is better than the example I followed above. :) Updated for explicit typing.